src/tests/EdFi.DataManagementService.Tests.E2E/Features/References/UpdateInvalidReferenceValidation.feature.disabled

```diff
@@ -2,31 +2,46 @@ Feature: Invalid Reference Validation
+                  | ["School"]                              | ["First grade"] | 255901107 | Grand Bend Elementary School |
+              And the system has these "programs"
+                  | programName                    | programTypeDescriptor                                                  | educationOrganizationReference     | programId |
+                  | Career and Technical Education | "uri://ed-fi.org/ProgramTypeDescriptor#Career and Technical Education" | {"educationOrganizationId":255901} | "100"     |
+              And the system has these Education Organization Category Descriptors
+                  | codeValue | namespace                                               | shortDescription |
+                  | School    | uri://ed-fi.org/EducationOrganizationCategoryDescriptor | School           |
+              And the system has these Students
+                  | studentUniqueId | birthDate  | firstName | lastSurname |
+                  | 604824          | 2010-01-13 | Traci     | Mathews     |
+              And the system has these Local Education Agencies
+                  | educationOrganizationCategoryDescriptor | localEducationAgencyId | localEducationAgencyCategoryDescriptor | nameOfInstitution |
+                  | ["School"]                              | 255901                 | ["Independent"]                        | Grand Bend ISD    |
+              And the system has these Student Education Organization Associations
+                  | educationOrganizationId | studentUniqueId |
+                  | 255901                  | 604834          |
+              And the system has these Student CTE Program Associations
+                  | beginDate  | educationOrganizationId | educationOrganizationId | programName                    | programTypeDescriptor              | studentUniqueId |
+                  | 2020-06-05 | 255901                  | 255901                  | Career and Technical Education | ["Career and Technical Education"] | 604825          |
+              And the system has these Graduation Plans
+                  | graduationPlanTypeDescriptor   | educationOrganizationId | schoolYear | totalRequiredCredits |
+                  | Career and Technical Education | 255901                  | 2022       | 10.000               |
+              And the system has these Student Section Associations
+                  | beginDate  | localCourseCode | schoolId  | schoolYear | sectionIdentifier           | sessionName             | studentUniqueId |
+                  | 2021-08-23 | ALG-1           | 255901001 | 2022       | 25590100102Trad220ALG112011 | 2021-2022 Fall Semester | 604874          |
 
         @ignore
         Scenario: 01 Ensure clients cannot update a resource with a Descriptor that does not exist
-             When a PUT request is made to "ed-fi/localEducationAgencies/d59e135ad9864e7392e737fd382c8607" with
+            Given Local Education Agency ID 255901
```

Comment:
I don't think these `Given` statements are necessary. We don't have any bindings that would satisfy them, and they are covered by the background section that sets up these entities.  I think you can remove all of them.